code/script.js

```diff
@@ -303,41 +351,83 @@ const filterCharacters = (keep) => {
         `No, the person doesn't wear ${attribute}! Remove all that wears ${attribute}`
       )
     }
-  } else if (group === 'other') {
-    // Similar to the one above
-  } else {
+  } else if (category === 'other') {
     if (keep) {
-      // alert popup that says something like: "Yes, the person has yellow hair! Keep all persons with yellow hair"
+      alert(
+        `Yes, the person has ${attribute}! Keep all that has ${attribute} `
```

Comment:
I am impressed you got the ${attribute} function to work, I had difficulties with this myself so call me impressed. :)